changedetectionio/notification.py

```diff
@@ -143,7 +144,7 @@ def process_notification(n_object, datastore):
         apobj = apprise.Apprise(debug=True, asset=asset)
         url = url.strip()
         if len(url):
-            print(">> Process Notification: AppRise notifying {}".format(url))
+            logger.debug(">> Process Notification: AppRise notifying {}".format(url))
```

Comment:
```suggestion
            logger.info(f">> Process Notification: AppRise notifying '{url}'")
```

Can be slightly 'higher' than 'debug'